Sources/SnapshotTesting/Snapshotting/UIImage.swift

```diff
@@ -32,6 +32,16 @@ extension Diffing where Value == UIImage {
       )
     }
   }
+  
+  
+  /// Used when the image size has no width or no height to generated the default empty image
+  private static func emptyImage() -> UIImage {
+    let label = UILabel(frame: CGRect(x: 0, y: 0, width: 200, height: 30))
+    label.backgroundColor = .red
+    label.text = "NO IMAGE GENERATED"
```

Comment:
@mackoj What do you think of enhancing this error message with more information? Something like: 
```suggestion
    label.text = "Error: No image could be generated for this view as its size was zero. Please set an explicit size in the test."
```
If you don't have time to make the adjustments, that's fine! We can merge as-is and try to make the changes ourselves soon.